File: Part2/C/BasicConcepts/BasicConcepts_Redeclaration.c
#include <stdio.h>

int main()
{
    // Do not redeclare variable names in most languages:
    char duplicateCharacter = 'a';
    char duplicateCharacter = 'b';
    
    printf ( "duplicateCharacter: %c\n", duplicateCharacter );
    
    return 0;
}

File: Part2/Rust/BasicConcepts/BasicConcepts_Redeclaration.rs
use std::char;
use std::i8;
use std::f32;

fn main() {
    // Do not redeclare variable names in most languages:
    let duplicateCharacter: char = 'a';
    let duplicateCharacter: char = 'b';
    println!("duplicateCharacter: {}", duplicateCharacter);
}